models/DataObject/Classificationstore/KeyGroupRelation.php

```diff
@@ -85,6 +85,22 @@ public static function create()
         return $config;
     }
 
+    /**
+     * @param int|null $keyId
+     * @param int|null $groupId
+     */
+    public static function getById($keyId = null, $groupId = null)
```

Comment:
Do we really need this? we already have `getByGroupAndKeyId` https://github.com/pimcore/pimcore/blob/10.x/models/DataObject/Classificationstore/KeyGroupRelation.php#L238

Reply:
You are right @dvesh3 - I removed the getById() but refactored `getByGroupAndKeyId()` to use the already existing https://github.com/pimcore/pimcore/blob/0b563345ac022a823e3bf16ac9fd1eafc72fd30f/models/DataObject/Classificationstore/KeyGroupRelation/Dao.php#L33-L47

Reply:
looks good to me. thanks!